source/server/options_impl_platform_default.cc

```diff
@@ -0,0 +1,9 @@
+#include <thread>
+
+#include "server/options_impl_platform.h"
+
+namespace Envoy {
+
+uint32_t OptionsImplPlatform::getCpuCount() { return std::thread::hardware_concurrency(); }
```

Comment:
Should we print a warning here that the platform does not support cpuset so Envoy is defaulting to HW concurrency?

Reply:
I think that's a good idea (since it's obviously not working as the user tries to use that). I added the warning.